floodviz/reference_parser.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+# open up and reference file and store the relevant data in a dictionary
+def parse_reference_data(path):
+    with open(path, 'r') as f:
+        data = json.load(f)
+
+        parsed_data = {}
+        parsed_data['epsg'] = data['target_epsg'][5:]
+        parsed_data['site_ids'] = data['site_ids']
+        parsed_data['display_sites'] = data['display_sites']
+        parsed_data['bbox'] = data['bbox']
+        parsed_data['start_date'] = data['startDate']
+        parsed_data['end_date'] = data['endDate']
+        parsed_data['peak_dv_date'] = data['endDate']
+        parsed_data['peak_site'] = data['peak']['site']
+
+        # grab the cities to be placed on the map
+        city_geojson_data = []
+        for i in range(0, len(data['reference']['features'])):
```

Comment:
Consider using a list comprehension with an if (https://docs.python.org/3/tutorial/datastructures.html?highlight=comprehension#list-comprehensions).I made this comment before but forgot to hit Add. Sorry about that. Same comment applies to rivers and background.